### ISSUE TITLE
how to use navmesh data that export from unity

### ISSUE THREAD

**Issue by liuhaopen** [recastnavigation/recastnavigation#372]
my game is develop with unity engine, and the server side use recastnavigation for AI navigation.
unity provide an API UnityEngine.AI.NavMesh.CalculateTriangulation to export navmesh data, so I wonder is there any way to create a dtNavMesh object by these triangle soup?

**Comment by monitor1394** [comment]
you can export unity scene to obj file for recastnavigation to build navmesh.for example,you can use [ExportSceneToObj](https://github.com/monitor1394/ExportSceneToObj) to export obj file.

**Comment by liuhaopen** [comment]
thanks for reply,I tried to export my scene to obj file and build navmesh in recastnavigation demo,but the results are somewhat different from unity export data(use same build param).
I think there's different build algorithm between unity and recastnavigation.So after exporting navmesh unity, I export the data via the UnityEngine.AI.NavMesh.CalculateTriangulation API,then convert to a dtNavMesh structure that can be used directly by recastnavigation.
I have implemented a rough version([Navigator](https://github.com/liuhaopen/Navigator "Navigator")), and I will improve it when I have time.
